src/MainWindow/main_window.cpp

```diff
@@ -167,7 +175,7 @@ void MainWindow::createActions() {
   });
 }
 
-void MainWindow::gui_start() { ReShowWindow("unknown"); }
```

Comment:
The idea of this change was to show everything at the start, all widget. Even if no project started.

Reply:
> The idea of this change was to show everything at the start, all widget. Even if no project started.

I'll change it back.